docs/02-app/02-api-reference/04-functions/cacheLife.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: cacheLife
+description: Learn how to use the cacheLife function to set the cache expiration time for a cached function or component.
+version: unstable
+related:
+  title: Related
+  description: View related API references.
+  links:
+    - app/api-reference/next-config-js/dynamicIO
+    - app/api-reference/directives/use-cache
+    - app/api-reference/functions/revalidateTag
+    - app/api-reference/functions/cacheTag
+---
+
+The `cacheLife` function is used to set tge cache lifetime of a function or component. It should be used alongside the `use cache` directive, and within the scope of the function or component.
+
+## Usage
+
+To use `cacheLife`, enable the [`dynamicIO` flag](/docs/app/api-reference/next-config-js/dynamicIO) in your `next.config.js` file and import `cacheLife` from `next/cache`.
+
+```ts filename="next.config.ts" switcher
+import type { NextConfig } from 'next'
+
+const nextConfig: NextConfig = {
+  experimental: {
+    dynamicIO: true,
+  },
+}
+
+export default nextConfig
+```
+
+```js filename="next.config.js" switcher
+const nextConfig = {
+  experimental: {
+    dynamicIO: true,
+  },
+}
+
+export default nextConfig
+```
+
+Then, invoke The `cacheLife` function within the scope of the function or component:
+
+```tsx filename="app/page.tsx" highlight={5} switcher
+'use cache'
+import { unstable_cacheLife as cacheLife } from 'next/cache'
+
+export default async function Page() {
+  cacheLife('hours')
+  return <div>Page</div>
+}
+```
+
+```tsx filename="app/page.tsx" highlight={5} switcher
+'use cache'
+import { unstable_cacheLife as cacheLife } from 'next/cache'
+
+export default async function Page() {
+  cacheLife('hours')
+  return <div>Page</div>
+}
+```
+
+## How cache revalidation works
```

Comment:
Hm, this should probably go into the deep dive as well

Reply:
Yeah, this doesnt feel like ref doc material. Good shout